packages/connect-react/src/components/append/AppendAfterHybridLoginScreen.tsx

```diff
@@ -0,0 +1,104 @@
+import log from 'loglevel';
+import React, { useCallback, useState } from 'react';
+
+import useAppendProcess from '../../hooks/useAppendProcess';
+import useShared from '../../hooks/useShared';
+import { AppendScreenType } from '../../types/screenTypes';
+import Checkbox from '../shared/Checkbox';
+import { LockIcon } from '../shared/icons/LockIcon';
+import { PasskeyAddIcon } from '../shared/icons/PasskeyAddIcon';
+import { LinkButton } from '../shared/LinkButton';
+import { Notification } from '../shared/Notification';
+import { PrimaryButton } from '../shared/PrimaryButton';
+
+const AppendAfterHybridLoginScreen = ({ attestationOptions }: { attestationOptions: string }) => {
+  const { config, navigateToScreen } = useAppendProcess();
+  const [dontShowAgain, setDontShowAgain] = useState(false);
+  const [error, setError] = useState<string | undefined>(undefined);
+  const [loading, setLoading] = useState(false);
+  const { getConnectService } = useShared();
+
+  const handleSubmit = useCallback(async () => {
+    if (loading) {
+      return;
+    }
+
+    setLoading(true);
+    setError(undefined);
+
+    const res = await getConnectService().completeAppend(attestationOptions);
+    if (res.err) {
+      log.error('error:', res.val);
+      setLoading(false);
+      setError('Passkey operation was cancelled or timed out.');
+
+      return;
+    }
+
+    if (dontShowAgain) {
+      const createEventRes = await getConnectService().recordEventUserAppendAfterCrossPlatformBlacklisted();
+
+      if (createEventRes?.err) {
+        log.error('error:', createEventRes.val);
+      }
+    }
+
+    setLoading(false);
+    navigateToScreen(AppendScreenType.Success);
+  }, [getConnectService, config, navigateToScreen, loading, attestationOptions]);
+
+  const toggleDontShowAgain = () => setDontShowAgain(prev => !prev);
+
+  const handleSkip = async () => {
+    if (dontShowAgain) {
+      const createEventRes = await getConnectService().recordEventUserAppendAfterCrossPlatformBlacklisted();
+
+      if (createEventRes?.err) {
+        log.error('error:', createEventRes.val);
+      }
+    }
+
+    config.onSkip();
+  };
+
+  return (
+    <div className='cb-append-after-hybrid-login-container'>
+      <div className='cb-h1'>Add a passkey to this device</div>
+      {error ? (
+        <Notification
+          className='cb-error-notification'
+          message={error}
+        />
+      ) : null}
+      <div className='cb-append-after-hybrid-login-icons'>
+        <PasskeyAddIcon className='cb-append-after-hybrid-login-icon' />
+      </div>
+      <div className='cb-append-after-hybrid-login-content'>
```

Comment:
![image](https://github.com/user-attachments/assets/032b895e-0081-42c0-a88e-d7bab4202480)
On my chrome the vertical alignments do not look good. Can we make sure that both the checkbox and the icon are vertically centered?